junit-runtime/src/main/scala/scala/scalanative/junit/Bootstrapper.scala

```diff
@@ -21,6 +21,7 @@ trait Bootstrapper {
   def before(instance: AnyRef): Unit
   def after(instance: AnyRef): Unit
 
+  def testMetadata(): TestClassMetadata
```

Comment:
Perhaps `testClassMetadata()` or `classMetadata()`, so that it's clear this applies to the class, not one test?

Reply:
Good idea, I've changed it to testClassMetadata